docs/device_model.md

```diff
@@ -0,0 +1,208 @@
+# Device Model
+
+This document describes the device model supported by `cloud-hypervisor`.
+
+## Summary
+
+| Device | Built-in | Build configurable | Enabled | Runtime configurable |
```

Comment:
Wow, this looks quite nice.
I'd remove the `Built-in` column and replace `Enabled` with `Enabled by default`.

Reply:
Ok that works for me!

Reply:
Done!